files/build_templates/qos_config.j2

```diff
@@ -265,7 +286,12 @@
 {% endfor %}
 {% for port in PORT_ACTIVE %}
         "{{ port }}|2": {
+        {% if port in port_names_list_extra_queues %}
```

Comment:
We'd better remove the leading spaces. Otherwise, they will be put ahead of the line `"scheduler"   : "[SCHEDULER|scheduler.1]",`, causing misalignment in the result json file.